windows-installer/run.py

```diff
@@ -0,0 +1,2 @@
+import subprocess
```

Comment:
If we keep this file and I'm correct in assuming that the name of the file is unimportant, can we give it a more descriptive name like `docs.py`?

Reply:
Not relevant anymore.